### PR TITLE
add: `pb-git`

### PR DESCRIPTION
**Pull request by vigress8**

### CODE REVIEW

File: packages/pb/pb.pacscript [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgver="0.0.1"
+pkgdesc="0x0 pastebin utility"
+maintainer="vigress8 <vig@disroot.org>"
+url="https://git.envs.net/envs/pb.git"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1 | cut -c1-8
+}
+
+prepare() {
+  git clone "${url}"
```
Comment (Elsie19) [inline packages/pb/pb.pacscript:11]: This isn't needed.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/pb/pb.pacscript (hunk 1 above)
Comment (vigress8) [reply packages/pb/pb.pacscript:11]: @Elsie19 when I remove `prepare()` I get this:

```
/tmp/pb.pacscript: line 11: cd: pb: No such file or directory
make: *** No rule to make target 'install'.  Stop.
	[!] ERROR: Could not package pb properly
```

File: packages/pb/pb.pacscript [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+package() {
+  cd "${name}"
```
Comment (Elsie19) [inline packages/pb/pb.pacscript:15]:
```suggestion
```

File: packages/pb/pb.pacscript (hunk 1 above)
Comment (Elsie19) [reply packages/pb/pb.pacscript:11]: remove it and apply the other suggestion I made.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/pb/pb.pacscript (hunk 1 above)
Comment (vigress8) [reply packages/pb/pb.pacscript:11]: @Elsie19 exact same `make` error:
`No rule to make target 'install'.  Stop.`

Comment (Elsie19) [reply packages/pb/pb.pacscript:11]: ok, I will look later.

File: packages/pb-git/pb-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name="pb-git"
+pkgver="0.0.1"
```
Comment (Elsie19) [inline packages/pb-git/pb-git.pacscript:2]:
```suggestion
pkgver="2022.11.03"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**